cvat-ui/src/actions/organization-actions.ts

```diff
@@ -91,6 +98,18 @@ const organizationActions = {
     updateOrganizationMemberFailed: (username: string, role: string, error: any) => createAction(
         OrganizationActionsTypes.UPDATE_ORGANIZATION_MEMBER_FAILED, { username, role, error },
     ),
+    getOrganizations: () => createAction(OrganizationActionsTypes.GET_ORGANIZATIONS, {}),
+    getOrganizationsSuccess: (array: Organization[], count: number, nextPageUrl?: string) => (
+        createAction(OrganizationActionsTypes.GET_ORGANIZATIONS_SUCCESS, { array, count, nextPageUrl })
+    ),
+    getOrganizationsFailed: (error: any) => createAction(OrganizationActionsTypes.GET_ORGANIZATIONS_FAILED, { error }),
+    updateOrganizationsGettingQuery: (query: Partial<OrganizationsQuery>) => (
```

Comment:
Do we really need this action? Usually we would use getOrganizations to setup the query

Reply:
We need this action because there are two components that rely on fetched organizations: the "Organization" menu item from the main header, and the organization selectors in the tasks/projects menus. So, if we don't keep data of the filters used to fetch a page of organizations, the list of organizations a user can switch between will become outdated